### PR TITLE
Add small refinements to homepage

### PR DESCRIPTION
**Pull request by kylefiedler**
screenshot 2015-10-02 11 53 44

### CODE REVIEW

File: app/assets/stylesheets/components/_repo-list.scss [hunk 1]
```diff
@@ -1,8 +1,9 @@
 .repo-list {
   display: flex;
   flex-wrap: wrap;
-  margin-right: -$base-spacing;
   padding-top: 1.25rem;
+  max-width: 100rem;
```
Comment (ColeTownsend) [inline app/assets/stylesheets/components/_repo-list.scss]: Isn't this going to be around ~1200px? I'd be worried that this is still too wide without some padding on the sides if the screen is near whatever 100rem comes out to.

File: app/assets/stylesheets/base/_variables.scss [hunk 2]
```diff
@@ -13,7 +13,8 @@ $heading-line-height: 1.2;
 $base-border-radius: 4px;
 $base-spacing: $base-line-height * 1rem;
 $small-spacing: $base-spacing / 2;
-$medium-spacing: ($small-spacing * 1.5);
+$medium-spacing: $small-spacing * 1.5;
```
Comment (ColeTownsend) [inline app/assets/stylesheets/base/_variables.scss]: Nice move on the incremental spacing! Looks great visually and it'll be easier later on.

**Comment by ColeTownsend** [comment]
Other than my width nitpick, looks good to me!

File: app/assets/stylesheets/components/_repo-list.scss [hunk 3]
```diff
@@ -11,11 +12,34 @@
...
-  position: relative;
+  transition: all 0.25s ease-out;
+
+  &:hover {
+    background-color: darken($base-background-color, 8%);
+    transition: all 0.25s ease-out;
+    transform: translateY(-3px);
+
+    &.high .repo-item-header {
+      background-color: shade($codetriage-red, 5%);
```
Comment (ryancoughlin) [inline app/assets/stylesheets/components/_repo-list.scss]: Nit pick, but could be a mixin if you wanted to.

**Comment by schneems** [comment]
Awesome!

**Pull request merged**